modules/standard/GPU.chpl

```diff
@@ -351,26 +351,33 @@ module GPU
     chpl_atomicTernOp(c_ptrTo(x), cmp, val);
   }
 
-  @chpldoc.nodoc
+  /* When run on a GPU, atomically add 'val' to 'x' (result is stored in 'x'). */
   inline proc gpuAtomicAdd(  ref x : ?T, val : T) : void { gpuAtomicBinOp("add", x, val); }
-  @chpldoc.nodoc
+  /* When run on a GPU, atomically subtract 'val' from 'x' (result is stored in 'x'). */
   inline proc gpuAtomicSub(  ref x : ?T, val : T) : void { gpuAtomicBinOp("sub", x, val); }
   @chpldoc.nodoc
```

Comment:
Deliberately not made public?

Reply:
Yes, from the OP:

> I now have doc comments for the various atomic functions. With the notable exception of gpuAtomicExch, which I don't think is one is too terribly useful unless it also returns the old value (which we don't do at the moment).

Not having these functions return values (as is done in their CUDA equivalents) was an oversight on my part when implementing these. I'm not sure if the return values are really that useful for the other operations (they return the old value before the atomic op) but for exchange it kind of seems useless without it.

Reply:
Knowing how to read is important. Thanks!